### PR TITLE
Add Avalanche Platform's C-Chain network

### PR DESCRIPTION
**Pull request by jpop32**
## Description

Add Avalanche Platform's C-Chain network

## Changes

Network added according to instructions found in: https://support.mycrypto.com/developers/adding-a-new-network

### CODE REVIEW

**Comment by jpop32** [comment]
Is there anything we can do to help speed this along? We're blocked on this for an integration with a 3rd party so it would be of great help to get this merged.

**Review by blurpesec — COMMENTED** [review]
> Is there anything we can do to help speed this along? We're blocked on this for an integration with a 3rd party so it would be of great help to get this merged.

Hey @jpop32 ! The only potential issue I see here is with the specified derivation path. The derivation path you specified contains the [slip-0044 code](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) `60` which is currently assigned to Ethereum. I just want to make sure that you're aware of that and that it won't cause an issue in the future

**Comment by jpop32** [comment]
Yes, that is intentional, C-Chain uses the same derivation path as Ethereum, for maximum compatibility.

File: src/database/data/networks.ts
Comment (blurpesec) [inline src/database/data/networks.ts:1039]: Can you change these to `AvalancheTestnet`? I'll approve after this :)

**Review by blurpesec — CHANGES_REQUESTED** [review]

File: src/database/data/networks.ts
Comment (jpop32) [reply src/database/data/networks.ts]: There you go.

**Review by blurpesec — APPROVED** [review]

File: src/database/data/networks.ts
Comment (blurpesec) [reply src/database/data/networks.ts]: Thanks!

**Comment by jpop32** [comment]
Thanks for the quick reaction! When could we expect to see this on the main website?

**Comment by FrederikBolding** [comment]
> Thanks for the quick reaction! When could we expect to see this on the main website?

Some time next week! We are deploying almost every day lately.

**Comment by blurpesec** [comment]
> > Thanks for the quick reaction! When could we expect to see this on the main website?
> 
> Some time next week! We are deploying almost every day lately.

Except Fridays :P

**Comment by jpop32** [comment]
Read only Fridays. I can respect that. :-)

**Comment by mussallem** [comment]
> > > Thanks for the quick reaction! When could we expect to see this on the main website?
> > 
> > 
> > Some time next week! We are deploying almost every day lately.
> 
> Except Fridays :P

Hi all.  Just wanted to check-in to see if there was some rough idea on timeframe for the next deployment so we could plan downstream on our end.  Thank you.

**Comment by FrederikBolding** [comment]
> > > > Thanks for the quick reaction! When could we expect to see this on the main website?
> > > 
> > > 
> > > Some time next week! We are deploying almost every day lately.
> > 
> > 
> > Except Fridays :P
> 
> Hi all. Just wanted to check-in to see if there was some rough idea on timeframe for the next deployment so we could plan downstream on our end. Thank you.

We're aiming to do a release tomorrow where this PR would be included 👍

**Comment by mussallem** [comment]
> > > > > Thanks for the quick reaction! When could we expect to see this on the main website?
> > > > 
> > > > 
> > > > Some time next week! We are deploying almost every day lately.
> > > 
> > > 
> > > Except Fridays :P
> > 
> > 
> > Hi all. Just wanted to check-in to see if there was some rough idea on timeframe for the next deployment so we could plan downstream on our end. Thank you.
> 
> We're aiming to do a release tomorrow where this PR would be included 👍

That is great.  Thank you!

**Comment by jpop32** [comment]
Just to confirm: network is visible on the site, I tried logging in with the Ledger, everything works as expected. Log in, balance, send, adding ERC-20s, sending ERC-20s, transaction history. Tried to connect with Metamask, that also works fine. Thanks!

One minor gripe: I have entered 470 as 'initial' value for network gas price in `networks.ts`, but the web defaults to 20. That is a potential issue for users, as 470 is currently the minimum accepted gas price, so unless they do 'show advenced features' on 'send' and set the price manually, the transactions won't be accepted. It would be wonderful if you could see why it doesn't default to 470. Other than that minor issue, everything is stellar.

**Comment by FrederikBolding** [comment]
> Just to confirm: network is visible on the site, I tried logging in with the Ledger, everything works as expected. Log in, balance, send, adding ERC-20s, sending ERC-20s, transaction history. Tried to connect with Metamask, that also works fine. Thanks!
> 
> One minor gripe: I have entered 470 as 'initial' value for network gas price in `networks.ts`, but the web defaults to 20. That is a potential issue for users, as 470 is currently the minimum accepted gas price, so unless they do 'show advenced features' on 'send' and set the price manually, the transactions won't be accepted. It would be wonderful if you could see why it doesn't default to 470. Other than that minor issue, everything is stellar.

This is in fact a bug, will fix in #3907. Thanks for reporting 😄

**Comment by FrederikBolding** [comment]
> > Just to confirm: network is visible on the site, I tried logging in with the Ledger, everything works as expected. Log in, balance, send, adding ERC-20s, sending ERC-20s, transaction history. Tried to connect with Metamask, that also works fine. Thanks!
> > One minor gripe: I have entered 470 as 'initial' value for network gas price in `networks.ts`, but the web defaults to 20. That is a potential issue for users, as 470 is currently the minimum accepted gas price, so unless they do 'show advenced features' on 'send' and set the price manually, the transactions won't be accepted. It would be wonderful if you could see why it doesn't default to 470. Other than that minor issue, everything is stellar.
> 
> This is in fact a bug, will fix in #3907. Thanks for reporting 😄

The bug has been fixed and a release will be made next week with the fix.

**Pull request merged**
